plugin/trino-postgresql/src/test/java/io/trino/plugin/postgresql/TestingPostgreSqlServer.java

```diff
@@ -50,7 +50,7 @@
     public TestingPostgreSqlServer()
     {
         // Use the oldest supported PostgreSQL version
-        dockerContainer = new PostgreSQLContainer<>("postgres:9.6")
+        dockerContainer = new PostgreSQLContainer<>("postgres:12")
```

Comment:
Please don't upgrade the version here. You ca introduce base test class and run tests on two versions likes  `BaseSqlServerConnectorSmokeTest`.

Reply:
`9.6` is already unsupported so I think it's the proper time to upgrade

Reply:
We would like to use old version here, so it should be `10.x` if you want to upgrade. The version 10 is enough for testing partitioned tables as far as I know.  
Also, please update `postgresql.rst` that has minimum required version.